pkg/minishift/registration/redhat.go

```diff
@@ -67,7 +74,12 @@ func (registrator *RedHatRegistrator) Register(param *RegistrationParameters) er
 			subscriptionCommand := fmt.Sprintf("sudo -E subscription-manager register --auto-attach "+
 				"--username %s "+
 				"--password '%s' ", param.Username, minishiftStrings.EscapeSingleQuote(param.Password))
+			spinnerView.Start()
+			startTime := time.Now()
 			_, err = registrator.SSHCommand(subscriptionCommand)
+			spinnerView.Stop()
+			fmt.Print("Registration successful ")
```

Comment:
@budhrg I believe you need to put a `\n` at the end of the Print statement and couple of dots before the statement e.g. fmt.Print("... Registration successful\n")